definitions/push-images/skopeo-copy-task.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: skopeo-copy-task
+spec:
+  params:
+    - name: srcImageURL
+      description: Source URL
+      type: string
+    - name: destImageURL
+      description: Destination URL
+      type: string
+    - name: srcToken
```

Comment:
Let's leave this here for now, but I think they will not be needed at some point (we'll use a service account in the PipelineRun).